nbviewer/providers/local/handlers.py

```diff
@@ -37,7 +37,10 @@ class LocalFileHandler(RenderingHandler):
 
     @property
     def localfile_path(self):
-        return os.path.realpath(self.settings.get('localfile_path', ''))
+        if self.settings.get('localfile_follow_symlinks'):
+            return os.path.abspath(self.settings.get('localfile_path', ''))
+        else:
+            return os.path.realpath(self.settings.get('localfile_path', ''))
```

Comment:
I think the logic here is inverted. `os.path.realpath` resolves (follows) symlinks to their targets (https://docs.python.org/3.6/library/os.path.html#os.path.realpath). `os.path.abspath` just gets the absolute path of the symlink itself. I think these need to be flipped, along with the code blocks below too.

Reply:
Ah, I see it now, that was an oversight on my end in the option renaming. Thanks for the review!